src/main/java/com/zeepy/server/community/repository/CommunityLikeRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.zeepy.server.community.repository;
+
+import com.zeepy.server.community.domain.CommunityLike;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CommunityLikeRepository extends JpaRepository<CommunityLike,Long> {
```

Comment:
```suggestion

@Repository
public interface CommunityLikeRepository extends JpaRepository<CommunityLike,Long> {
```
Repository어노테이션을 달아주세요.